src/main.py

```diff
@@ -647,7 +635,7 @@ def replica_return(self, db_state, zk_state):
             # Wal receiver is not running and
             # postgresql isn't in archive recovery
             # We should try to restart
-            logging.warning('We should try switch primary one more time here.')
+            logging.warning('We should try switch source of WAL to {}'.format(holder))
```

Comment:
`source of WAL` is a non-existing term.
Should we really introduce it?
Why not `We should try switch primary to {} again`?